MensaHub-Junction/src/main/java/de/olech2412/mensahub/junction/gui/components/own/boxes/MealBox.java

```diff
@@ -69,7 +70,20 @@ public MealBox(String mealName, String description, String price, String allerge
 
     public void showRecommendation(PredictionResult predictionResult){
         long predictionScore = Math.round(predictionResult.getPredictedRating());
-        recommendationBadge.setText("Deine Empfehlung: " + predictionScore + "/5" + " | Genauigkeit: " + predictionResult.getTrustScore());
+        recommendationBadge.setText("Empfehlung: " + predictionScore + "/5" + " | " + predictionResult.getTrustScore());
         recommendationBadge.addClassName("prediction");
+        Tooltip tooltip = Tooltip.forComponent(recommendationBadge) // important, dont remove!
+                .withText("Diese Angaben wurden für dich persönlich auf Grundlage deiner und den Bewertungen anderer Nutzer für dich berechnet. " +
+                        "Die Angabe der Genauigkeit (" + predictionResult.getTrustScore() + ")" + " gibt an, wie wahrscheinlich die Prognose ist.")
+                .withPosition(Tooltip.TooltipPosition.TOP_START);
+        tooltip.setManual(isMobileDevice());
+        recommendationBadge.addClickListener(spanClickEvent -> {
```

Comment:
ClickListener needed because tooltip does not work for touch devices by default. so we check if its a mobile device, set tooltip to manual and react on the click